samples/subsys/event_manager/sample.yaml

```diff
@@ -0,0 +1,13 @@
+sample:
+  description: Sample showing Event Manager usage
+  name: Event Manager sample
+tests:
+  samples.event_manager:
+    build_only: true
+    arch_exclude: xtensa
+    integration_platforms:
+      - nrf52dk_nrf52832
```

Comment:
use something more generic here,e.g. `qemu_x86`. It is typically used in zephyr (or `native_posix`)